wordpress-mobile/WordPress-Android
# Show post status on post list search results

Adds post status to search results on post list - based on [this discussion](https://github.com/wordpress-mobile/WordPress-Android/pull/11291#issuecomment-589876505).

To test:
1. Open Post List
2. Set author filter to "Me"
3. Verify the rows contain just date
4. Set author filter to "Everyone"
5. Verify the rows contain date and author name
6. Click on the search icon and type something
7. Notice the search results contain date, name and post status

* Perform the above steps in both compact and full view


| Before | After
|------|-------|
| ![Screenshot_1591609598](https://user-images.githubusercontent.com/2261188/84039441-87b77f00-a9a1-11ea-84e7-76f111ac9f4b.png) | ![Screenshot_1591609363](https://user-images.githubusercontent.com/2261188/84039456-8c7c3300-a9a1-11ea-8b53-473af57d56da.png) |



PR submission checklist:


@SylvesterWilmott Could you please take a quick look at this change. Thanks!
